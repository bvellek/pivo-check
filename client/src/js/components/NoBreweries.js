import React from 'react';

const NoBreweries = () => (
  <section className="no-cities">
    <p>Looks like there are no breweries here! Check back later, our database of breweries is updated&nbsp;frequently.</p>
  </section>
);

export default NoBreweries;
