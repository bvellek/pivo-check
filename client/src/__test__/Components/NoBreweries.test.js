import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NoBreweries from '../../js/components/NoBreweries';

describe('<NoBreweries />', () => {
  it('renders without crashing', () => {
    shallow(<NoBreweries />);
  });
  it('it renders correctly', () => {
    const component = shallow(
      <NoBreweries />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
