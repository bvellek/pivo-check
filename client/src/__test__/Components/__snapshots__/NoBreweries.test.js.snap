exports[`<NoBreweries /> it renders correctly 1`] = `
<section
  className="no-cities">
  <p>
    Looks like there are no breweries here! Check back later, our database of breweries is updatedÂ frequently.
  </p>
</section>
`;
